### PR TITLE
Deprecate close_comm_on_error.

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Deprecate close_comm_on_error.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
https://github.com/home-assistant/core/pull/99946

- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by joostlek** [comment]
Do we still bother to keep this field in if it isn't going to be used anyway?

**Comment by janiversen** [comment]
@joostlek I would like to keep it documented until we remove it. It serves as an extra reminder and tell the user that the parameter exist. If we remove it too fast, while there are still many configurations using it, it might cause issues about missing documentation.

I am not sure if the standard have changed, but last I deprecated parameters that was the way to do it.

**Comment by joostlek** [comment]
I don't have much knowledge or ideas about this so I'll wont review this PR and leave it to sometone who does know :)

**Comment by janiversen** [comment]
Fair ! thanks for your comment.

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -38,9 +38,7 @@ The modbus integration provides a number of parameters to help communicate with
 
 {% configuration %}
 close_comm_on_error:
```
Comment (frenck) [inline source/_integrations/modbus.markdown]: If something is deprecated, we remove it from the documentation (with one exception: if the whole integration is deprecated, we place a warning box).

As the latter is not the case here, please remove this one.

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (janiversen) [reply source/_integrations/modbus.markdown]: ok will do tomorrow.

Comment (frenck) [reply source/_integrations/modbus.markdown]: Hehe you have an odd tomorrow 😉 

Thanks for updating! ❤️ 

../Frenck

**Review by frenck — APPROVED** [review]

**Pull request merged**